dashboard/app/models/teacher_score.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: teacher_scores
+#
+#  id            :integer          not null, primary key
+#  user_level_id :integer
+#  teacher_id    :integer
+#  score         :integer
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+# Indexes
+#
+#  index_teacher_scores_on_created_at     (created_at)
+#  index_teacher_scores_on_teacher_id     (teacher_id)
+#  index_teacher_scores_on_user_level_id  (user_level_id)
```

Comment:
what queries are you planning to run against this table? 

you should only need to index columns which will appear in the where clause of a query, so I think you can get rid of the index on `created_at`.

we'll definitely need to query against `user_level_id`, so that index can stay. but I don't see a need to index on `teacher_id`, so that should be removed until we need it, unless you see a need for it now.

Reply:
Got it - I removed the index from teacher_id and created_at and left it on user_level_id.